core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -37,6 +42,9 @@
       + "error response entities.";
   protected static final boolean DEBUG_CONFIG_DEFAULT = false;
 
+  public static final Set<String> HEADER_CONFIG_ACTIONS =
+          initializedHeaderConfigActions();
```

Comment:
Consider doing this inline so that a reader doesn't need to jump around the code to find the value:
```
private static final Set<String> HEADER_CONFIG_ACTIONS = Utils.mkSet("set", "add", "setDate", "addDate");
```